packages/dataviews/src/utils/use-debounced-input.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { useDebounce } from '@wordpress/compose';
+
+export default function useDebouncedInput( defaultValue = '' ) {
```

Comment:
Would it make sense to have this hook in `@wordpress/compose` since it's used by multiple packages.

Reply:
Yes it would now that is used in 3 packages. I'll do it in a follow up right after this one though.